### ISSUE TITLE
The bones multiply over the others in Synfig Studio 3.11pre2

### ISSUE THREAD

**Issue by Gilraiser** [synfig/synfig#714]
*Synfig Studio 3.11-pre2 in Archlinux 64:

Issue description:

After saving and closing a file with bones. When the file is opened again, the bones multiply over the others.It also happens that when opening files created with previous versions of Synfig Studio, there are errors in the animation made with the skeleton deformation layer.

![bone--00212](https://user-images.githubusercontent.com/45057390/51781599-c28c8000-20e8-11e9-83d1-3dab5b7ad9d4.jpg)
With Synfig Studio 2.2.2 using skeleton deformation layer
![plano-10-2](https://user-images.githubusercontent.com/45057390/51781727-e355d500-20ea-11e9-83de-857396ea2804.png)
With Synfig Studio 3.11-pre2 using skeleton deformation layer
![plano-10](https://user-images.githubusercontent.com/45057390/51781732-efda2d80-20ea-11e9-91fe-ec1035bd22c6.png)

**Comment by morevnaproject** [comment]
Yes, the two files at before and after state would help a lot! ^__^

**Comment by Gilraiser** [comment]
Hello, thanks for answering. the first file is created in Synfig Studio 2.1.1
[girlofblue.zip](https://github.com/synfig/synfig/files/2825894/girlofblue.zip)
The second one is the same but modified in Synfig 3.11pre2
[RM-Escena-01-Plano-01.zip](https://github.com/synfig/synfig/files/2825895/RM-Escena-01-Plano-01.zip)
The file of the dog is made in Synfig 2.2.2. But to open it in 3.11pre2 is that the error arises.

[whitedog.zip](https://github.com/synfig/synfig/files/2825876/whitedog.zip)

Many thanks

**Comment by morevnaproject** [comment]
The issue with skeleton deformation is fixed by @blackwarthog in 8093129d59aafab971e6592d569ccdf1900c62ad

**Comment by Gilraiser** [comment]
Well, thank you very much.

**Comment by Gilraiser** [comment]
Hello, I hope you are very well. I want to send you a better example of the error that causes some bones to double, since the example in .sif that I sent you before was apparently not the best. I hope this information helps the project. Beforehand thank you very much.

This is the initial file. 
[example-01.sif.zip](https://github.com/synfig/synfig/files/3537588/example-01.sif.zip)

This is after copying it to another file.
[example-02.sifz.zip](https://github.com/synfig/synfig/files/3537589/example-02.sifz.zip)

Here I attach a video demonstration.
[](https://www.youtube.com/watch?v=aTBuhxQpS1c&feature=youtu.be)

**Comment by rodolforg** [comment]
I believe this was partially fixed by #1438

**Comment by Gilraiser** [comment]
Many thanks
